.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
Willen we dit aanpassen? Alles files zijn nu CRLF.

Reply:
dit zijn instellingen uit slimme apparaten

Reply:
als je die wil aanpassen goed